src/app/pages/edit-contact/edit-contact.component.html

```diff
@@ -0,0 +1,93 @@
+<h1>Edit Contact: {{ contact.name }}</h1>
+
+<form class="contact-form">
+  <div>
+    <label for="address">Address:</label>
+  </div>
+  <input
+    type="text"
+    name="address"
+    value="{{ contact.address }}"
```

Comment:
Setting the value isn't necessary, the ngModel should handle that. This goes for all the inputs below too.